_posts/2020-11-04-ggplot2.md

```diff
@@ -0,0 +1,92 @@
+---
+name: Scott Auerbach
+title:  "Intro to ggplot2 and its parameters"
+date:   2020-11-04
+---
+The goal of this blogpost is to introduce the reader to the powerful tool `ggplot2`
+and provide an explanation and scope of some of the abilities of this tool to create effective
+visualizations of experimental data. By reading this post, the reader will be
+able to use the parameters laid out and write code of their own. Any set of tabular data can be sorted, analyzed and visualized with `ggplot2`.
+
+## General Format
+
+`ggplot2` has three basic components: the data set, which is best provided in
+*.csv* format because it cannot be edited accidentally or intentionally, the
```

Comment:
I believe .csv formats can still be edited. Maybe highlight instead the value of using a plain-text file format like .csv for its universal readability?